Kconfig

```diff
@@ -2,6 +2,27 @@ mainmenu "SOF $(PROJECTVERSION) Configuration"
 
 comment "Compiler: $(CC_VERSION_TEXT)"
 
+menu "CAVS"
+
+config HP_MEMORY_BANKS
+	int "HP memory banks count"
+	depends on CAVS
+	default 8
+	help
+	  Available memory banks count for High Performace memory
+	  Lowering avalible banks could result in lower power consumption
```

Comment:
available

Reply:
@dabekjakub it's also missing the block sizes IIRC 64k, this makes the number of blocks option more meaningful.